MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TabControl.xaml

```diff
@@ -199,59 +198,73 @@
-        <Setter Property="Padding" Value="16"/>
+        <Setter Property="Padding" Value="0"/>
         <Setter Property="Height" Value="72" />
         <Setter Property="Width" Value="72" />
+        <Setter Property="Margin" Value="4"/>
         <Setter Property="wpf:RippleAssist.Feedback" Value="{DynamicResource MaterialDesignFlatButtonRipple}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type TabItem}">
-                    <Grid x:Name="Root" >
-                        <wpf:ColorZone x:Name="tabitemcz"
-                                       HorizontalAlignment="Stretch"
-                                       VerticalAlignment="Stretch"
-                                       Focusable="False"
-                                       Mode="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Mode)}" 
-                                       Background="{x:Null}"
-                                       wpf:ColorZoneAssist.Background="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Background)}"
-                                       wpf:ColorZoneAssist.Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Foreground)}">
-                            <wpf:Ripple Focusable="False"
-                                        ClipToBounds="True"
-                                        Content="{TemplateBinding Header}" 
-                                        ContentTemplate="{TemplateBinding HeaderTemplate}"
-                                        ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
-                                        ContentStringFormat="{TemplateBinding HeaderStringFormat}"
-                                        HorizontalContentAlignment="Center"
-                                        VerticalContentAlignment="Center"
-                                        RecognizesAccessKey="True"
-                                        x:Name="contentPresenter"
-                                        Opacity=".82"
-                                        SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" 
-                                        Padding="{TemplateBinding Padding}"
-                                        TextBlock.FontWeight="Medium"
-                                        TextBlock.FontSize="15"
-                                        TextOptions.TextFormattingMode="Ideal" 
-                                        TextOptions.TextRenderingMode="Auto">
-                            </wpf:Ripple>
-                        </wpf:ColorZone>
+                    <Grid x:Name="Root"
+                          Cursor="Hand">
+                        <Grid >
+                            <Ellipse Fill="{TemplateBinding Foreground}"
```

Comment:
I realize the button states on the spec shows the items as circles, however the spec does not call out that all items in the rail should display circles when hovered/clicked. Even within Google's implementations on Android/Gmail/Youtube, it is more common to have the hover/click look fill the space as a square rather than a circle. I think for now the default implementation should stick with that shape.